src/tarantool_schema.c

```diff
@@ -557,6 +572,25 @@ tarantool_schema_get_sid_by_string(
 	return space->space_number;
 }
 
+/* ALIAS for verify existens */
```

Comment:
What do you mean? That we can check whether the space exists by `sid` using this function?

Reply:
Removed misleading comment.